include/dp_maglev.h

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and IronCore contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __INCLUDE_DP_MAGLEV_H__
+#define __INCLUDE_DP_MAGLEV_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include <stdint.h>
+#include "dp_util.h"
+#include "dp_lb.h"
+
+int dp_delete_maglev_backend(struct lb_value *lbval, const uint8_t *delete_server);
+int dp_add_maglev_backend(struct lb_value *lbval, const uint8_t *new_server);
```

Comment:
I'm a bit confused as to what `server` means in this context, it seems to suggest to me that this is somehow distributed. Or it's just the source material nomenclature leftover. We use `back_ip` everywhere in place of this, unless I am misunderstanding how this works.

Reply:
Changed the names to back_ip to be consistent with the rest of the code.